### PR TITLE
Improvements in SubCommand_CleanGhost.java

### PR DESCRIPTION
**Pull request by Chris6ix**
Can you please explain "Starting checking the shop be ghost, all does not exist shop will be removed..." a little better...not sure what it means... (line 57)
What do you mean by "posting" (to main server thread)? (line 110)
What exactly is meant by "disallow create the shop"? (line 117) (Maybe that its not allowed to create a shop in this location?)

### CODE REVIEW

**Comment by Ghost-chu** [comment]
debugLog only for developers so all logs can be skip

File: src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public void onCommand(
 
         MsgUtil.sendMessage(sender,
                 ChatColor.GREEN
-                        + "Starting checking the shop be ghost, all does not exist shop will be removed...");
+                        + "->Starting checking the shop be ghost, all does not exist shop will be removed...");
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java:58]: This line is notice send to server owner (no translable string) and all not exists shop will be removed (container block missing)

File: src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java [hunk 2]
```diff
@@ -107,14 +107,14 @@ public void onCommand(
                 // Shop exist check
                 Util.mainThreadRun(() -> {
                     Util.debugLog(
-                            "Posted to main server thread to continue accessing Bukkit API for shop "
+                            "->Posted to main server thread to continue accessing Bukkit API for shop "
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java:110]: Debug log, ignore it

File: src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java [hunk 3]
```diff
@@ -107,14 +107,14 @@ public void onCommand(
...
                                     + shop);
                     if (!Util.canBeShop(shop.getLocation().getBlock())) {
                         MsgUtil.sendMessage(sender,
                                 ChatColor.YELLOW
                                         + "Deleting shop "
                                         + shop
-                                        + " because the target location is nolonger a shop or disallow create the shop.");
+                                        + " ->because the target location is nolonger a shop or disallow create the shop.");
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java:117]: Untranlsable string that sends to server owner (shop deleted cause shop in disallowed region)

**Review by Ghost-chu — COMMENTED** [review]
Reviewed

**Comment by Chris6ix** [comment]
Ready to merge

**Pull request merged**